common/src/main/scala/explore/components/AgGridHelpers.scala

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components
+
+import scala.annotation.unused
+
+import cats._
+import cats.data.Validated._
+import cats.syntax.all._
+import cats.effect.IO
+import crystal.ViewF
+import crystal.react.implicits._
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.string._
+import explore.AppCtx
+import explore._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.MonocleReact._
+import lucuma.core.util.Display
+import lucuma.core.util.Enumerated
+import lucuma.ui.forms._
+import lucuma.ui.optics._
+import monocle.Lens
+import monocle.macros.Lenses
+import org.scalajs.dom.document
+import org.scalajs.dom.ext.KeyCode
+import org.scalajs.dom.html
+import react.semanticui.elements.button.Button
+import reactST.agGridCommunity.colDefMod._
+import reactST.agGridCommunity.iCellEditorMod.ICellEditorParams
+import reactST.agGridCommunity.iCellRendererMod.ICellRendererParams
+import reactST.agGridReact.components._
+
+import scalajs.js
+
+object AgGridHelpers {
+
```

Comment:
I haven't been able to figure out a way to make the public methods typesafe. The renders, etc. are doing a lot of `asInstanceOf[View[...]]`s, but I can't ensure that these are the correct types.